x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/details_page/index.tsx

```diff
@@ -48,7 +50,13 @@ export const AgentConfigDetailsPage: React.FunctionComponent = () => {
   const [redirectToAgentConfigList] = useState<boolean>(false);
   const agentStatusRequest = useGetAgentStatus(configId);
   const { refreshAgentStatus } = agentStatusRequest;
+  const {
+    application: { navigateToApp },
+  } = useCore();
+  const routeState = useIntraAppState<AgentConfigDetailsDeployAgentAction>();
   const agentStatus = agentStatusRequest.data?.results;
+  const openEnrollmentFlyoutOpenByDefault =
+    queryString.parse(useLocation().search).openEnrollmentFlyout === 'true';
```

Comment:
Can we use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here, instead?

```suggestion
  const queryParams = new URLSearchParams(useLocation().search);
  const openEnrollmentFlyoutOpenByDefault = queryParams.get('openEnrollmentFlyout') === 'true';
```

Reply:
FYI - If I remember correctly, the last time I tried to use URLSearchParams I had issues in the Jest UT environment because I think `jsdom` did support it. I don't believe we have UT for these UIR components, so it maybe find to use it for now

Reply:
I just pulled in the `URLSearchParams` as opposed to `query-string` and yes this doesn't run in Jest UT right now afaik.  I plan to use functional tests when our host tests are re-enabled, so we'll see what happens.